frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -495,12 +494,46 @@ export class TileViewPage extends React.Component {
   }
 
   componentDidMount() {
-    const { items, filterGroups, getAvailableFilters, groupByTypes } = this.props;
+    const {
+      items,
+      filterGroups,
+      getAvailableFilters,
+      groupByTypes,
+      storeFilterKey,
+      filterRetentionPreference,
+      defaultActiveFilters,
+    } = this.props;
     const { categories } = this.state;
     const availableFilters = getAvailableFilters(defaultFilters, items, filterGroups);
-
+    if (!(storeFilterKey && localStorage.getItem(storeFilterKey)) && defaultActiveFilters) {
+      _.each(Object.keys(defaultActiveFilters), (filterGroup) => {
+        _.each(defaultActiveFilters[filterGroup], (key) => {
+          availableFilters[filterGroup][key].active = true;
+        });
+      });
+    }
     const activeValues = getActiveValuesFromURL(availableFilters, filterGroups, groupByTypes);
 
+    if (storeFilterKey && localStorage.getItem(storeFilterKey)) {
```

Comment:
Filters set via URL take precedence over filters from the local storage. They should not be additive.

If there are no filters from the URL, only then apply those from local storage.

Reply:
okay, I am implementing such that user preference(local storage) is only applied when filters are not added in URL.
P.S: URL keywords are additive and apply conjunction with the default active Filters